packages/core/database/package.json

```diff
@@ -36,6 +36,7 @@
     "fs-extra": "10.0.0",
     "knex": "1.0.7",
     "lodash": "4.17.21",
+    "semver": "7.3.8",
```

Comment:
Do we really need semver for this ?

Reply:
I find it is the safest way. For cases like `7.3.8` < `7.3.11` where it's `true` but a simple string comparison would be `false`. Same for when we will have MySQL 11 (`11.0.0` < `8.0.0`):p
We can do without but then we need to do string manipulation

Reply:
I'm more worried DB versions might not respect this convention instead. but this will work for now.